Foundation/NSHost.swift

```diff
@@ -14,8 +14,19 @@ import Darwin
 import Glibc
 #endif
 
+import Foundation
```

Comment:
This is incorrect; it will cause Darwin builds to fail since it will attempt to import the objc Foundation. Please remove this import